### PR TITLE
docs: Add steps to convert a Standalone MongoDB container to a Replica Set 

### PR DESCRIPTION
**Pull request by AnaghHegde**
> Add steps to convert stand alone to clustered set up for mongodb containers in server set up guide.

### CODE REVIEW

File: contributions/ServerSetup.md [hunk 1]
```diff
@@ -70,6 +70,16 @@ Note that this command doesn't set any username or password on the database so w
 
 MongoDB will now be running on `mongodb://localhost:27017/appsmith`.
 
+###Convert a Standalone to a Replica Set
+- Upgrade the MongoDB version to 4.4 or higher if it’s 4.2 or lower (https://www.mongodb.com/docs/manual/release-notes/4.4/#std-label-4.4-upgrade)
+- Close the mongoDB instance running in your local
+- Start the mongoDB in replica set mode and initiate the replica set
+    - mongod --port 27017 --dbpath <path/to/db> --replSet <replica-set-name> && mongo --eval “rs.initiate()”
```
Comment (abhvsn) [inline contributions/ServerSetup.md:77]: Please add the steps to run mongo with replica set from docker.

File: contributions/ServerSetup.md [hunk 2]
```diff
@@ -102,7 +112,7 @@ cp envs/dev.env.example .env
 
 This command creates a `.env` file in the `app/server` folder. All run scripts pick up environment configuration from this file.
 
-5. Ensure that the environment variables `APPSMITH_MONGODB_URI` and `APPSMITH_REDIS_URI` in the file `.env` point to your local running instances of MongoDB and Redis.
+5. Ensure that the environment variables `APPSMITH_MONGODB_URI` and `APPSMITH_REDIS_URI` in the file `.env` point to your local running instances of MongoDB and Redis. And make sure to update the replica set name with correct value in the mongo connection string.
```
Comment (abhvsn) [inline contributions/ServerSetup.md:115]:
```suggestion
5. Ensure that the environment variables `APPSMITH_MONGODB_URI` and `APPSMITH_REDIS_URI` in the file `.env` point to your local running instances of MongoDB and Redis. Also please make sure to update the replica set name with correct value in the mongo connection string. 
```

File: contributions/ServerSetup.md [hunk 3]
```diff
@@ -70,6 +70,16 @@ Note that this command doesn't set any username or password on the database so w
 
 MongoDB will now be running on `mongodb://localhost:27017/appsmith`.
 
+###Convert a Standalone to a Replica Set
```
Comment (abhvsn) [inline contributions/ServerSetup.md:73]:
```suggestion
### Convert a standalone MongoDB node to a replica set
```

**Review by abhvsn — COMMENTED** [review]
Added very minor comments.

File: contributions/ServerSetup.md (hunk 1 above)
Comment (AnaghHegde) [reply contributions/ServerSetup.md:77]: @abhvsn I don't understand this.

**Review by mohanarpit — APPROVED** [review]

**Pull request closed without merging**